src/components/Camera/style.css

```diff
@@ -103,6 +102,21 @@
   /*Mirroring transform: scale(-1, 1); */
 }
 
+@supports (object-fit: cover){
+  .video{height: 100%; width:100%}
+}
+/* this will only apply styles to Edge (and IE)*/
+@supports (object-fit: cover) and (-ms-ime-align:auto) {
+  .video {
+    height:100%;
+    width:auto;
+    @media (--small-viewport) {
+      /* HACK: this will make the video look centered on small screens */
+      left: -50%;
```

Comment:
This seems... Dangerous? What happens on screens just before the `--small-viewport` width is hit

Reply:
Large screen 

<img width="529" alt="screen shot 2018-07-12 at 10 23 26" src="https://user-images.githubusercontent.com/7127427/42624653-cc79ced6-85bd-11e8-9095-6d423dcc49d9.png">

Small screen

<img width="415" alt="screen shot 2018-07-12 at 10 22 11" src="https://user-images.githubusercontent.com/7127427/42624657-ced88a82-85bd-11e8-810d-dc720663128a.png">

Reply:
Okay, opening this up in the inspector, I think you're depending on a coincidence here.
`width: auto` makes the video be as wide as its normal resolution (which is normally much wider than the screen).

I guess for your device `left: -50%` must _happen_ to get it to centre. If you go mess around with the screen size a bit, you'll see that you've not actually centred it - you've just offset it.

Unless I'm misunderstanding the scope of the `auto` width, and what the screen resolutions could be 🤔 

Does just going for `width: 100vw` or something not work? It looks like that works, and [Edge does support it](https://caniuse.com/#feat=viewport-units)

Reply:
Yes, it does support it, but if I use `width: 100vw` I will end up with the same problem we are trying to solve. See screenshot. I know using `left: -50%` is a hack and probably won't work everywhere but I don't know how else to fix this problem. I also tried giving the parent a defined height and width but it didn't make any difference...
<img width="430" alt="screen shot 2018-07-13 at 10 57 06" src="https://user-images.githubusercontent.com/7127427/42685661-8682cc9c-868b-11e8-89b7-14d29f50b5f8.png">

Reply:
Ahh, sorry - I misunderstood.

In that case, why not set the width/height of the video with absolute values, from the JS side using the window width/height (& update on screen size change)?

It's also a hack, but a cleaner hack imo, and would be more functionally accurate